src/tools/sdk/_windows.md

```diff
@@ -1,23 +1,41 @@
+###How to install?
+
 You can install the Dart SDK using [Chocolatey.][Chocolatey]
 
-{{site.alert.important}}
-These commands require administrator privileges.
-If you need help on starting an administrator-level command prompt,
-try a search like
-<em><a href="https://www.google.com/search?q=cmd+admin"
-  target="blank">cmd admin</a>.</em>
-{{site.alert.end}}
+> **_NOTE:_** These commands require administrative privileges. To start command prompt as an administrator following these steps:
```

Comment:
I think the previous `{{site.alert.important}}` structure matches the rest of the website better, but if you think it's highlighted too much you can use `{{site.alert.note}}` instead which introduces the Note text and icon.

Reply:
Ok i'll try that.